crates/astria-composer/local.env.example

```diff
@@ -27,6 +27,9 @@ ASTRIA_COMPOSER_API_LISTEN_ADDR="0.0.0.0:0"
 # Address of the RPC server for the sequencer chain
 ASTRIA_COMPOSER_SEQUENCER_URL="http://127.0.0.1:26657"
 
+# Chain ID of the sequencer chain which transactions are submitted to.
+ASTRIA_COMPOSER_SEQUENCER_CHAIN_ID="astria-dev-1"
```

Comment:
IMO remove in favor of querying sequencer's `/genesis` endpoint to extract the cometbft chain ID from there. This is how conductor is doing it and I would like this to be consistent between services.